guides/how-we-do-user-research.md

```diff
@@ -64,6 +64,75 @@ These are the things that user researchers usually do on projects.
+  * YYYY MM DD - CLIENT - PROJECT - Discussion Guide - RESEARCH ACTIVITY
+  * For example, 2019 04 08 - HM Passport Office - Renewals Beta - Discussion Guide - Usability Testing
+
+And a consistent front cover with:
+  * Client name
+  * Project name
+  * Date
+  * Main contact (who wrote it)
+
+### The parts of a discussion guide
+
+#### Introduction
+Recap the goals of the project and the research activity, describe the methods you’ll be using, and set out the overall structure of the research session.
+
+#### Preparation
+Describe anything the researcher needs to have or do, to prepare for the session. Reference things like the information sheet to send to participants, describe how to set up the prototype you’ll be testing, and reference the [notes template](https://drive.google.com/drive/folders/1MAUax08jwwqNd1BWfH-gixSbltY2qE-m) you want to use.
+
+#### Welcome
+Set out the things you'll say to the participant to let them them know what’s going on and make sure they feel comfortable. It can be helpful to write this out word for word, so you have something to fall back on when you’re tired, or get interrupted or distracted.
+
+The welcome should include things like:
+* **Thank you** - start off by letting participants know you appreciate them offering their time (incentivised or not)
+* **Introduce yourself** - name, role, company and who you are working for
+* We are **independent researchers**, working on behalf of a client
+* **Signed consent/NDA** (if applicable) - has this been done? Any questions?
+* **No right or wrong answers**, we’re only interested in what the participant thinks and the influences behind that
+* **Talk aloud as you go** - easier said than done, but this is incredibly important when testing prototypes of existing services
+* **Be honest** - as researchers we won’t be offended by any comments, we just want to find out what people really think about a service and how it might be improved
+* **Research purpose** - Recap what the session will focus on (without giving away too much detail that may alter behaviour) e.g. We’re interested in finding out more about why and how you would contact organisations like Ofsted or, we’re trying to learn more about how you would complete “x” task.
+* **Session set up** - who is taking notes or observing the session, recording set up, how long the session will take, what we’ll be doing in that time (e.g. general discussion, tasks, card sorting etc.) so that participants know what to expect.
+* **Remind them it is voluntary** - they can choose to stop at any time.
+
+#### Topics and activities
+Have a section in your guide for each of the main interview topics, test tasks or workshop tasks.
+
+For each one write out:
+* What you are trying to learn
+* For an interview topic, the starter questions you’ll ask along with likely follow up topics
+* For a task, the steps to do, and any instructions and explanations for participants
+* How long the topic or task is likely take
```

Comment:
"How long the topic or task is likely take" - missing "to"

Reply:
Thanks. Fixed.